File: src/array/python3/1_TwoSum.py
#
# @lc app=leetcode id=1 lang=python3
#
# [1] Two Sum
#

# @lc code=start
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(0,len(nums)-1,1):
            for j in range(i+1,len(nums),1):
                two_sum=nums[i]+nums[j]
                if two_sum==target:
                    return [i,j]
        
# @lc code=end

File: work/typescript/array/1_TwoSum.ts
/*
 * @lc app=leetcode id=1 lang=typescript
 *
 * [1] Two Sum
 */

// @lc code=start
function twoSum(nums: number[], target: number): number[] {
    for( let i: number = 0; i < nums.length-1; i++){
        for(let j :number = i+1; j < nums.length; j++){
            if(nums[i]+nums[j] == target){
                return [i,j]
            }
        }
    }
};
// @lc code=end

